packages/salesforcedx-vscode-core/src/panels/orgListPanel.ts

```diff
@@ -302,6 +303,19 @@ export class OrgListPanel {
                   // Now display the the content (the panel and the grids)
                   var orgListContentEl = document.getElementById('org-list-content');
                   orgListContentEl.style.display = 'contents';
+
+                  setTimeout(() => {
```

Comment:
Forgive me.  This is nasty.  The dom is not updated immediately, so a slight timeout allowed me to query and find the elements i wanted to change the style of.  But this is nothing but a short-term hack.

Reply:
Oh, that's ugly.  (but it made me laugh) Yea, we need to get this resolved b/f shipping.  I opened a ticket up on the issue at https://github.com/microsoft/vscode-webview-ui-toolkit/issues/378

Reply:
I'd love to hack on this some more if ananya wants to move forward.